README.md

```diff
@@ -52,6 +52,28 @@ var Smooch = require('smooch');
 Smooch.init({appId: '<app-id>'});
 ```
 
+## Browser support
+
+Web Messenger supports all popular browsers. 
+
+#### Desktop versions
+
+Chrome: Latest and one major version behind
```

Comment:
Should these be markdown bullets? Otherwise it looks like this:

![screen shot 2017-10-03 at 11 15 19 am](https://user-images.githubusercontent.com/2432382/31133001-3dd5c97e-a82c-11e7-8056-b00d685eccc5.png)